modules/upgrade-duration-rhel-nodes.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-upgrade-duration.adoc
+
+:_content-type: CONCEPT
+[id="redhat-enterprise-linux-nodes_{context}"]
+= {op-system-base-full} nodes
+
+{op-system-base-full} worker nodes require an additional usage of OpenShift-Ansible to update node binary components. This rolling update is similar to the MCO initiated rolling update and therefore it has a similar duration. The time it takes to reboot a particular node varies significantly. In cloud instances, the reboot might take about 1-2 minutes whereas in physical bare metal hosts the reboot might take more than 15 minutes.
```

Comment:
> whereas in physical bare metal hosts the reboot might take more than 15 minutes.
Per https://github.com/openshift/openshift-docs/pull/46241#discussion_r897740326, RHOCS reboot may take 5 mins and we didn't get an estimation on RHEL reboot. @gpei @pamoedom What's your opinion on this part, does `15 mins` look good to you?

Reply:
As Pedro mentioned in https://github.com/openshift/openshift-docs/pull/46241#discussion_r897740326, the reboot time of physical bare metal hosts is highly variable as different machine models may have different boot checks, and also affected by the BIOS settings, iPXE boot timeout, etc, so I'm afraid we can't give a general estimation time for all the physical machines. 
How about saying "in physical bare metal hosts the reboot might take much longer and variable as it depends on the machine models"